CONTRIBUTING.md

```diff
@@ -15,14 +15,11 @@ These keywords `SHALL` be in `monospace` for ease of identification.
 This repository uses [`pre-commit`]. If you add a new filetype, you `SHOULD` add
 a new [hook][pre-commit hook].
 
-From the repository root directory:
-
-```sh
-source src/sh/pre-commit.sh
-```
-
 See the `cfg/` directory for assorted formatter and linter configurations.
 
+To install our `pre-commit` hooks locally and/or run our python formatter
+scripts, please see the [python hooks readme].
```

Comment:
Overall I think this documentation is great, thanks for the initiative on this

Though I do think that the README in the PR would better be suited in `cfg/README.md`

Because the content you've written there is not specific to *just* Python, there's actually a lot of content about setting up `pre-commit` just generally